### ISSUE TITLE
MobileNet-V2 SSD: Is there a  script to test a model and which returns the mAP value and some other metrics ?

### ISSUE THREAD

**Issue by antocad** [tensorflow/models#10102]
Hi, I would like to know if there is a way to test a model (from a .tfrecord file or .jpg) and which returns the mAP value and some other metrics ?

Thank you.

**Comment by Nannigalaxy** [comment]
You can get mAP, P-R curve on the validation set with [this](https://github.com/Cartucho/mAP).

**Comment by jvishnuvardhan** [comment]
@Nannigalaxy Thanks for the answer. @antocad Can you please check the above answer and let us know whether it is resolved or not. 

If not resolved, can you please share a simple standalone code to reproduce the issue. Thanks!

**Comment by antocad** [comment]
Yes it has resolved my issue! Thanks @Nannigalaxy for your answer :)
